### PR TITLE
feat: add kafka notifier

### PR DESCRIPTION
**Pull request by yordis**
This allows consuming events without needing to write Go code or use slack/email.

### CODE REVIEW

File: internal/notify/kafka.go [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+	"gocloud.dev/pubsub"
+)
+
+type Kafka struct {
+	publisher *pubsub.Topic
+	cfg       *service.KafkaConfig
+}
+
+func NewKafka(logger log.Logger, cfg *service.KafkaConfig) (*Kafka, error) {
```
Comment (adamdecaf) [inline internal/notify/kafka.go:25]: This needs to be in multi.go as well. Was the intention to publish events on a new topic?

File: internal/notify/kafka.go [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+	publisher, err := kafka.OpenTopic(logger, cfg)
+	if err != nil {
+		return nil, err
+	}
+
+	return &Kafka{publisher: publisher, cfg: cfg}, nil
+}
+
+type event struct {
```
Comment (adamdecaf) [inline internal/notify/kafka.go:34]: Did we want to specify an event type? So multiple message types could be sent on the topic?

**Comment by adamdecaf** [comment]
That CI failure is on us. We shouldn't be trying to push images on forks.

File: internal/notify/kafka.go (hunk 1 above)
Comment (yordis) [reply internal/notify/kafka.go:25]: We would provision a given topic where we would publish the notification events. So we do need a new topic.

File: internal/notify/kafka.go (hunk 2 above)
Comment (yordis) [reply internal/notify/kafka.go:34]: Probably? Ideally, types everywhere, just in case.

**Pull request merged**